client/src/document/index.tsx

```diff
@@ -111,12 +111,16 @@ export function Document(props /* TODO: define a TS interface for this */) {
         </Suspense>
       )}
 
-      <nav className="breadcrumb-locale-container">
+      <div className="breadcrumb-container">
         {doc.parents && <Breadcrumbs parents={doc.parents} />}
+      </div>
+
+      <div className="locale-container">
```

Comment:
nit-picking: I'd put a language select into `<nav>` and ditch the `<div>`'s.

Reply:
Hmmm, not sure whether it can be termed navigation. It kinda navigates to a different locale, but the same page. Definitely, something to consider. The one thing we would need to do if we change this to a `nav` element is to ensure we label all the various `nav` containers so it is clear what the purpose of each `nav` landmark is.